carbon-design-system/ibm-security
# Removed flex that has been added to .bx--link which collapses the whi…

…te space around the words

## Pull request - Removed the inline-flex for .bx--link class to avoid no spacing issue

### Affected issues

- Resolves #

### Proposed changes

- 

### Testing instructions

- Go to the IBM Security components and then navigate to Shell component. You should see the Header "IBM Security Application" with proper spacing.
